code/__DEFINES/flags.dm

```diff
@@ -95,3 +95,7 @@ GLOBAL_LIST_INIT(bitflags, list(1, 2, 4, 8, 16, 32, 64, 128, 256, 512, 1024, 204
 #define ACID_PROOF 32 //acid stuck on it doesn't melt it.
 #define INDESTRUCTIBLE 64 //doesn't take damage
 #define FREEZE_PROOF 128 //can't be frozen
+
+//Voting flags
+#define WEIGHTED 1

```

Comment:
I'd add a way to make custom votes weighted, but I don't know how to do checkboxes in spessmen HTML